src/github.rs

```diff
@@ -13,45 +13,177 @@ use crate::{database::Database, github::{
     pull_requests::PullRequestsRepositoryPullRequestsNodesReviewRequestsNodesRequestedReviewer::User,
 }, settings::Repository as RepoInfo};
 
+pub(crate) use pull_requests::{PullRequestState, PullRequestReviewState};
+
 const GITHUB_FETCH_SIZE: i64 = 10;
 const GITHUB_URL: &str = "https://api.github.com/graphql";
 const APP_USER_AGENT: &str = concat!(env!("CARGO_PKG_NAME"), "/", env!("CARGO_PKG_VERSION"),);
 const INIT_TIME: &str = "1992-06-05T00:00:00Z";
 
-type DateTime = String;
+pub(crate) type DateTime = chrono::DateTime<Utc>;
 
+#[allow(clippy::upper_case_acronyms)]
+type URI = String;
 #[derive(GraphQLQuery)]
 #[graphql(
     schema_path = "src/github/schema.graphql",
     query_path = "src/github/issues.graphql",
-    response_derives = "Debug"
+    response_derives = "Debug",
+    scalar = "DateTime = DateTime<Utc>"
```

Comment:
It seems that `scalar = "DateTime = DateTime"` would be more appropriate. Could you check this?